dll/win32/iernonce/iernonce.h

```diff
@@ -0,0 +1,10 @@
+/*
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     ReactOS Extended RunOnce processing with UI.
+ * COPYRIGHT:   Copyright 2021 He Yang <1160386205@qq.com>
+ */
+
+#pragma once
+
+typedef VOID(CALLBACK *RUNONCEEX_CALLBACK)(UINT CompleteCnt, UINT TotalCnt, DWORD Reserved);
```

Comment:
It would be actually better to move all that into a `iernonce_undoc.h` file inside `sdk/include/reactos/` or something...
(together with the prototypes for `InitCallback` and `RunOnceExProcess`).
And instead, keeping this `iernonce/iernonce.h` file here to be used as a precompiled header.

Reply:
should we also generate some files like `librnonce.lib` so people can link to it statically?

Reply:
Not sure whether it's of importance right now.

Reply:
In general, only add that when it's needed.
Otherwise we have a ton of unused .lib files being built for no good reason, which will only make compilation take longer